README.md

```diff
@@ -1,4 +1,7 @@
 🎂 Depreday
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
 ===========
 
 Now you have the change to commemorate important deprecation day marks
```

Comment:
`the chance`*

Reply:
This is an automated PR, I'll fix it in the next one (together with this messy layout)